packages/mdl-snackbar/mdl-snackbar.scss

```diff
@@ -0,0 +1,131 @@
+/**
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import "mdl-animation/mixins";
+@import "mdl-theme/mixins";
+@import "mdl-typography/mixins";
+@import "./variables";
+
+/* stylelint-disable scss/dollar-variable-pattern, scss/at-mixin-pattern */
+@mixin _mdl-snackbar-reflexive-margin($default-side, $value, $reset-opposite-margin: true) {
+  @if (index((right, left), $default-side) == null) {
+    @error "Invalid default margin side #{default-side}. Please specifiy either right or left";
+  }
+
+  $rtl-side: left;
+
+  @if ($default-side == left) {
+    $rtl-side: right;
+  }
+
+  margin-#{$default-side}: $value;
+  [dir="rtl"] & {
+    margin-#{$rtl-side}: $value;
+
+    @if ($reset-opposite-margin) {
+      margin-#{$default-side}: 0;
+    }
+  }
+}
+
+/* stylelint-enable scss/dollar-variable-pattern, scss/at-mixin-pattern */
+
+/* postcss-bem-linter: define snackbar */
+.mdl-snackbar {
+  display: block;
+  display: flex;
+  position: fixed;
+  bottom: 0;
+  left: 50%;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: flex-end;
+  padding-right: 24px;
+  padding-left: 24px;
+  transform: translate(0, 100%);
+  transition: transform .25s $mdl-animation-fast-out-linear-in-timing-function;
+  background-color: $mdl-snackbar-background-color;
+  will-change: transform;
+  pointer-events: none;
+
+  @media (max-width: ($mdl-snackbar-tablet-breakpoint - 1)) {
+    left: 0;
+    width: calc(100% - 48px);
+  }
+
+  @media (min-width: $mdl-snackbar-tablet-breakpoint) {
+    min-width: 288px;
+    max-width: 568px;
+    transform: translate(-50%, 100%);
+    border-radius: 2px;
+  }
+
+  &--active {
+    transform: translate(0, 0);
+    pointer-events: auto;
+    transition: transform .25s $mdl-animation-linear-out-slow-in-timing-function;
+
+    @media (min-width: $mdl-snackbar-tablet-breakpoint) {
+      transform: translate(-50%, 0);
+    }
+  }
+
+  &__text {
+    @include mdl-typography(body1);
+    @include _mdl-snackbar-reflexive-margin(right, auto, true);
+    display: flex;
+    align-items: center;
+    height: 48px;
+    color: white;
```

Comment:
supernit: you think since we factored out the bg color into a variable, we should do the same here?